webapp/src/errorList.tsx

```diff
@@ -44,7 +44,7 @@ export class ErrorList extends React.Component<ErrorListProps, ErrorListState> {
             <div className="errorListInner" hidden={this.state.isCollapsed}>
                 {
                     (this.state.errors || []).map(e =>
-                        <div key={errorKey(e)}>{e.messageText}</div>)
+                        <div key={errorKey(e)}>{`${e.messageText} - [line ${e.line + 1}: col ${e.column}]`}</div>)
```

Comment:
Looks great! Nice compact change

Reply:
This is a good start and we can always tweak how things appear as we go

Reply:
Unless we plan to translate line and col, you should remove them.

Reply:
should this display `e.column + 1` like it does for for `e.line`? Just checked vs code and it gives both line and column indexed from 1 ~